internal/mapping.go

```diff
@@ -142,6 +142,58 @@ func ToSpannerForeignKey(conv *Conv, srcFkName string) string {
 	return getSpannerValidName(conv, srcFkName)
 }
 
+// ToSpannerOnDelete maps the source ON DELETE action
+// to the corresponding Spanner compatible action.
+// The following mapping is followed:
+// a) CASCADE/NO ACTION -> mapped to the same as source action
+// b) all others -> NO ACTION (default)
+//
+// Since only MySQL and PostgreSQL have this functionality
+// as of yet, for other sources OnDelete fields are
+// kept empty i.e. "" is mapped to ""
+//
+// For all source actions converted to a different action,
+// an issue is appended to it's TableLevelIssues to
+// generate a warning message for the user
+func ToSpannerOnDelete(conv *Conv, srcTableId string, srcDeleteRule string) string {
+	srcDeleteRule = strings.ToUpper(srcDeleteRule)
+	if srcDeleteRule == "NO ACTION" || srcDeleteRule == "CASCADE" || srcDeleteRule == "" {
```

Comment:
In MYSQL `RESTRICT` and `NO ACTION` mean the same thing. 

Reference - https://stackoverflow.com/questions/39338474/whats-the-different-between-restrict-and-no-action-in-mysql

`RESTRICT` in MySQL should also get mapped to `NO ACTION` in Spanner,

Reply:
All actions other than CASCADE are mapped to NO ACTION